README.md

```diff
@@ -88,6 +98,8 @@ sudo usermod -aG docker $USER
    conda env create -f environment.yml
    ```
 
+   > **_NOTE:_** In case of missing permissions for writing packages, run: `sudo chown -R $USER ~/.conda`
+
 3. Update if necessary the `docker-compose.yml` file to your project paths and conda environment name.
```

Comment:
Actually, this step could be moved up to 1.

Reply:
but this way it comes after the environment creation, and since you need the proper name this seems safer? Maybe we should add a line about how to retrieve the name and env location